pkg/api/v1/types.go

```diff
@@ -1249,6 +1249,22 @@ type PodSpec struct {
 
 // PodSecurityContext holds pod-level security attributes and common container settings.
 type PodSecurityContext struct {
+	// Capabilities are the capabilities to add/drop when running containers
+	Capabilities *Capabilities `json:"capabilities,omitempty"`
+
+	// Run all containers in privileged mode
+	Privileged *bool `json:"privileged,omitempty"`
+
+	// SELinuxOptions is the SELinux context to be applied to all containers
+	SELinuxOptions *SELinuxOptions `json:"seLinuxOptions,omitempty"`
+
+	// RunAsUser is the UID to run the entrypoint of all container processes
+	RunAsUser *int64 `json:"runAsUser,omitempty"`
+
+	// RunAsNonRoot indicates that all containers should be run as a non-root user.
+	// If the RunAsUser field is not explicitly set then the kubelet may check the
```

Comment:
please also fix comments on the old SecurityContext - default values are all wrongly documented